apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -1888,6 +1915,7 @@ export async function grade(submission, variant, question, question_course) {
       );
 
       if (Object.keys(resultData.format_errors).length > 0) resultData.gradable = false;
+
       return {
         courseIssues,
         data: {
```

Comment:
Nit: Is the explicit listing intentional / needed? This code could be shorter without an explicit re-definition of everything from `resultData` to `data`.

Reply:
This could work `const { thingIdontneed, ...data } = result`

Reply:
This is done this way for two reasons:

- Things like `options` and `variant_seed` aren't needed by callers; it's better not to expose that to them.
- Some things must be renamed, e.g. we use `submitted_answers` and `correct_answers` internally but they must be `submitted_answer` and `true_answer` in the return value.